typescript/src/entities/domain.ts

```diff
@@ -0,0 +1,31 @@
+import { Entity } from '../entity';
+
+export class Domain extends Entity {
+  protected static resourceName: string = "domains";
+  protected static singularName: string = "domain";
+  protected static pluralName: string = "domains";
+
+  @Domain.property("id")
+  private _id?: number;
+
+  get id(): number | undefined {
+    return this._id;
+  }
+
+  set id(newId: number | undefined) {
+    this._id = newId;
+    this._isDirty = true;
+  }
+
+  @Domain.property("domain")
```

Comment:
nothing serious but a property pretty much needs us to write 10 lines of code, this is pretty sad to me, any better alternative exists?

Reply:
already discussed previously: https://github.com/merchisdk/sdk/pull/181